node/src/api/server.rs

```diff
@@ -1,78 +1,25 @@
 use crate::{
     api::{middleware::*, routing},
     config::NodeConfig,
-    consensus::ConsensusProcessor,
     db::utils::db::build_pool,
 };
 use actix_cors::Cors;
 use actix_web::{http, middleware::Logger, web, App, HttpResponse, HttpServer};
 use serde_json::json;
-use std::{net::ToSocketAddrs, sync::mpsc};
+use std::net::ToSocketAddrs;
```

Comment:
It looks like I disabled the consensus workers in this last merge. Sorry I had done a rebase and messed it up but had thought I correctly corrected it. Will fix in my testing PR going up today.